content/akeneo-web-api.yaml

```diff
@@ -796,6 +796,30 @@ paths:
         404:
           $ref: "#/responses/404Error"
 
+  /api/rest/v1/product-models/{code}:
+    get:
+      summary: Get a product model
+      operationId: "get_product_models__code_"
+      tags:
+        - Product models
+      x-versions:
+        - 2.0.1
+      description: This endpoint allows you to get the information about a given product model.
+      parameters:
+        - $ref: '#/parameters/code'
+      responses:
+        200:
+          description: OK
+          x-details: Returns the content of the product model in JSON standard format.
+          schema:
+            $ref: '#/definitions/ProductModel'
+        401:
+          $ref: "#/responses/401Error"
+        406:
```

Comment:
Still missing the 403 error

Reply:
My bad. You are right. No 403 here since there are neither ACLs nor EE permissions applied for the moment.